tnth/prod-anon.yaml

```diff
@@ -0,0 +1,33 @@
+services:
```

Comment:
right, just wasn't sure we should displace the other .yaml in the same directory, `system-services.yaml` .. so i thought forcing the user to name it might be best.

if that makes deployment difficult, happy to rename.